lib/utils/utils.go

```diff
@@ -14,20 +14,22 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/ChainSafe/chaindb"
-	"github.com/dgraph-io/badger/v2"
+	"github.com/ChainSafe/gossamer/internal/database"
 	"github.com/stretchr/testify/require"
 )
 
 // DefaultDatabaseDir directory inside basepath where database contents are stored
 const DefaultDatabaseDir = "db"
 
 // SetupDatabase will return an instance of database based on basepath
-func SetupDatabase(basepath string, inMemory bool) (*chaindb.BadgerDB, error) {
-	return chaindb.NewBadgerDB(&chaindb.Config{
-		DataDir:  filepath.Join(basepath, DefaultDatabaseDir),
-		InMemory: inMemory,
-	})
+func SetupDatabase(basepath string, inMemory bool) (database.Database, error) {
+	nodeDatabaseDir := filepath.Join(basepath, DefaultDatabaseDir)
+	return database.NewPebble(nodeDatabaseDir, inMemory)
+}
+
+func ClearDatabase(basepath string) error {
+	nodeDatabaseDir := filepath.Join(basepath, DefaultDatabaseDir)
+	return os.RemoveAll(nodeDatabaseDir)
```

Comment:
can you move this to the `database` package? Rather not continue to add to this `utils` package given this package doesn't conform to go package naming best practices.